superset-frontend/src/views/App.tsx

```diff
@@ -43,26 +49,39 @@ const bootstrap = JSON.parse(container?.getAttribute('data-bootstrap') ?? '{}');
 const user = { ...bootstrap.user };
 const menu = { ...bootstrap.common.menu_data };
 const common = { ...bootstrap.common };
+let lastLocation: string;
```

Comment:
can we call this `lastLocationPathname` or `lastPathname` to make it more clear?